templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+      <p class="p-heading--four">Cloud-native edge clusters for on premise intelligence</p>
+      <ul>
+        <li>Bring 5G networks and edge clouds to production</li>
+        <li>Deploy AI/ML solutions in industrial settings</li>
+        <li>Get latest versions and security fixes with over-the-air updates</li>
+        <li>Offload IT infrastructure management to our experts</li>
+        <li>Contract long term support services for mission critical software</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
+      
+        <li class="p-inline-images__item u-no-margin--left">
+          <a href="https://microstack.run/">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/6a983826-Openstack+Logo.svg",
+                alt="Openstack",
+                height="88",
+                width="102",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          <a href="https://microk8s.io/">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/9309d097-MicroK8s_SnapStore_icon.svg",
+                alt="Microk8s",
+                height="88",
+                width="88",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          <a href="https://linuxcontainers.org/lxd/introduction/">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/d85ef015-image-lxd.svg",
+                alt="LXD",
+                height="88",
+                width="95",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+      </ul>          
+      <a href="/openstack">
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+  <!-- Shop floor cobots -->
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2>Shop floor cobots</h2>
+      <p class="p-heading--four">Speed, precision and quality</p>
+      <ul>
+        <li>Assure robots security over the long term</li>
+        <li>Benefit from a managed cloud infrastructure for robots-as-a-service</li>
+        <li>Boost developer productivity on ROS and ISAAC</li>
+        <li>Leverage cutting-edge accelerated computer vision libraries</li>
+        <li>Run deep learning inference engines, natively or in containers</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
```

Comment:
I think `p-inline-images--small` matches the design slightly closer and looks better.